Documentation/spelling_wordlist.txt

```diff
@@ -1137,6 +1137,7 @@ vCPU
 vCPUs
 vX
 validateCNPs
+validatingAdmissionPolicy
```

Comment:
Hi, and thanks!
You will need to rebase your PR on the main branch, there are conflicts to address.

You can now drop this change when doing so.